src/pages/dtrader/_our-platforms.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import mt5Icon from '../../images/svg/dtrader/mt5.svg'
+import goIcon from '../../images/svg/dtrader/go-icon.svg'
+import ezIcon from '../../images/svg/dtrader/ez-icon.svg'
+import dxIcon from '../../images/svg/dtrader/dx-icon.svg'
+import dbIcon from '../../images/svg/dtrader/db-icon.svg'
+import GenericCarousel from 'components/custom/carousel/_platforms-carousel'
+
+const OurPlatforms = () => {
+    const carouselData = [
```

Comment:
please move this out, I've seen the same component several times now, can we please make this a reusable component or something?

Reply:
Yes we have a plan to make it reusable also somewhere I added todo comment as well for this component. 
If we have a time then we can work on it. 

Thank @mohsen-deriv for reviewing 🙏 

@habib-deriv